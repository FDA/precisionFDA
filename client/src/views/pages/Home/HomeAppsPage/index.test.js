import React from 'react'
import { shallow } from 'enzyme'

import { HomeAppsPage } from './index'


describe('HomeAppsPage test', () => {
  it('should render', () => {
    const component = shallow(<HomeAppsPage />)

    expect(component).toMatchSnapshot()
  })
})