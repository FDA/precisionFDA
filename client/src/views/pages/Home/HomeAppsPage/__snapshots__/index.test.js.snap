// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeAppsPage test should render 1`] = `
<HomeLayout>
  <div
    className="home-page-layout__header-row"
  >
    <div
      className="home-page-layout__actions"
    >
      <Link
        to="/home/apps/me"
      >
        <Button
          type="primary"
        >
          <span>
            <Icon
              icon="fa-cubes"
            />
            Â  Create App
          </span>
        </Button>
      </Link>
    </div>
    <div
      className="home-page-layout__actions--right"
    >
      <DropdownMenu
        title="Actions"
      />
    </div>
  </div>
  <div
    className="pfda-padded-t20"
  >
    <Connect(HomeAppsTable)
      apps={Array []}
    />
  </div>
</HomeLayout>
`;
